### PR TITLE
added LoginContainer components for issue23

### PR DESCRIPTION
**Pull request by drybell**
LoginContainer is the main wrapper for the login in components. Components are broken down to Header, Footer, and Form. Form encapsulates the username/password text fields and the sign in button component.

### CODE REVIEW

File: src/frontend/src/components/LoginContainer/Footer/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './Footer.css';
```
Comment (duci9y) [inline src/frontend/src/components/LoginContainer/Footer/Footer.tsx]: This file is empty.

File: src/frontend/src/components/LoginContainer/Form/Button/Button.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './Button.css';
```
Comment (duci9y) [inline src/frontend/src/components/LoginContainer/Form/Button/Button.tsx]: CSS is empty again.

File: src/frontend/src/components/LoginContainer/Form/Form.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+          <div className="e-mail">
+            <p className="info">E-Mail</p>
+            <input type="text" name="e-mail" />
+          </div>
+          <div className="password">
+            <p className="info">Password</p>
+            <input type="password" name="pass" />
+          </div>
+          <label>
+            <Button />
```
Comment (duci9y) [inline src/frontend/src/components/LoginContainer/Form/Form.tsx]: If you're not passing any props to this, it doesn't need to be a separate component, unless you expect custom functionality in the future.

File: src/frontend/src/components/LoginContainer/Form/Form.tsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+          <p> Sign In </p>
+          <div className="e-mail">
+            <p className="info">E-Mail</p>
+            <input type="text" name="e-mail" />
+          </div>
+          <div className="password">
+            <p className="info">Password</p>
+            <input type="password" name="pass" />
+          </div>
+          <label>
```
Comment (duci9y) [inline src/frontend/src/components/LoginContainer/Form/Form.tsx]: Why wrap a button in a label?

File: src/frontend/src/components/LoginContainer/LoginContainer.tsx [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './LoginContainer.css';
+import Header from './Header/Header';
```
Comment (duci9y) [inline src/frontend/src/components/LoginContainer/LoginContainer.tsx]: If there is an index.tsx exporting the component, you don't need to dig deep into `./Header/Header`, `./Header` would work.

File: src/frontend/src/components/LoginContainer/Form/Form.tsx [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+    <div className="Form">
+      <form className="Login">
+        <div className="form-group">
+          <p> Sign In </p>
+          <div className="e-mail">
+            <p className="info">E-Mail</p>
+            <input type="text" name="e-mail" />
+          </div>
+          <div className="password">
+            <p className="info">Password</p>
```
Comment (duci9y) [inline src/frontend/src/components/LoginContainer/Form/Form.tsx]: If you are not using CSS, don't prematurely use CSS classes that may not be correct anymore/in the future.

File: src/frontend/src/components/LoginContainer/Header/Header.tsx [hunk 7]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+// import './Header.css';
+import tutv_logo from './tutv-logo.svg';
+
+const Header: React.FC = () => {
+  return (
+    <div className="header">
+      <img src={tutv_logo} className="TUTV-logo" alt="tutv-logo" />
```
Comment (duci9y) [inline src/frontend/src/components/LoginContainer/Header/Header.tsx]: Remove CSS classnames if there's no CSS

File: src/frontend/src/components/LoginContainer/Form/Button/Button.tsx [hunk 8]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+// import './Button.css';
+
+const Button: React.FC = () => {
+  return (
+    <div className="submit-button">
+      <button
+        type="submit"
+        className="button-primary"
+        // onClick={()}
```
Comment (nivi-nath) [inline src/frontend/src/components/LoginContainer/Form/Button/Button.tsx]: remove commented code from final merge

File: src/frontend/src/components/LoginContainer/Form/Form.tsx [hunk 9]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+// import './Form.css';
```
Comment (nivi-nath) [inline src/frontend/src/components/LoginContainer/Form/Form.tsx]: here too

File: src/frontend/src/components/LoginContainer/Header/Header.tsx [hunk 10]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+// import './Header.css';
```
Comment (nivi-nath) [inline src/frontend/src/components/LoginContainer/Header/Header.tsx]: commented code

File: src/frontend/src/components/LoginContainer/LoginContainer.tsx [hunk 11]
```diff
@@ -0,0 +1,18 @@
...
+import Footer from './Footer/Footer';
+import Form from './Form/Form';
+// import Button from "./Button/Button"
```
Comment (nivi-nath) [inline src/frontend/src/components/LoginContainer/LoginContainer.tsx]: commented code

**Pull request closed without merging**